addOns/jxbrowsers/jxbrowser/CHANGELOG.md

```diff
@@ -60,3 +61,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - First version
 
+[13]: https://github.com/zaproxy/zap-extensions/releases/jxbrowser-v13
```

Comment:
Are these footer links going to work? Are we tagging things differently going forward? (I tried this, which I assume doesn't work because the release doesn't exist yet. Then I tried n-1 links and they 404 as well ... ex: https://github.com/zaproxy/zap-extensions/releases/jxbrowser-v12)

Reply:
Yes, the plan is to start tagging the add-on releases.